docs/en/Xamarin.Essentials/SensorSpeed.xml

```diff
@@ -47,7 +47,7 @@
       </ReturnValue>
       <MemberValue>3</MemberValue>
       <Docs>
-        <summary>Get the sensor data as fast as possible.</summary>
+        <summary>Get the sensor data as fast as possible (the atual rate is platform-dependent).</summary>
```

Comment:
"the actual"

Reply:
Thanks for catching this one, James! Fixed with b6ea7e6e3beefcf0079bee1512f4565d42efcb1e.